### ISSUE TITLE
Автообновление модулей

### ISSUE THREAD

**Issue by ivanbalashov239** [uzyovoys/aggregate#6]
Для начала руками закинуть не сложно, но в дальнейшем хотелось бы обновлять автоматом, можно автоматически билдить из репозитория, но лучше бы наверное обновление бинарных сборок силами агрегата сделать, а на будущее плагин менеджер, понятное дело.

**Comment by morfeusys** [comment]
Лучше магазин наподобие скриптов в Дусе или, чего уж там, App Store или Google Play ))

**Comment by morfeusys** [comment]
Обновления могут потребовать новые конфиги кстати

**Comment by ivanbalashov239** [comment]
@Morfeusys Ок, магазин понятное дело в перспективе, но просьба предусмотреть консольный вариант, хотя можно, конечно, делать пакеты для системного пакетного менеджера.
ну конфиги понятно, не беда.

**Comment by morfeusys** [comment]
Консольный вариант уже работает. Просто в документации для него не нашлось пока времени )) Там консоль вообще довольно обширная. Если заметили - в конфиге есть упоминание про shell.

**Comment by ivanbalashov239** [comment]
я наоборот решил, что это не работающая функция, при первом запуске(без модулей) попробовал подключиться получил в выводе кракозябры, отложил на потом.

**Comment by morfeusys** [comment]
Нет, там вполне можно или телнетом на порт 5000 зайти, или в браузере набрать что-то типа http://lcoalhost:8080/shell.html

**Comment by ivanbalashov239** [comment]
@Morfeusys да, сглупил, с телнетом лучше :+1:

**Comment by Merlin74** [comment]
Так функция shell не только локально или доступ только там где запущен агрегат? Если первое то как-то не безопасненько.

**Comment by ivanbalashov239** [comment]
@Merlin74 безопасность в данном случае на совести пользователя, если вы в вашем фаерволе откроете эти порты наружу, то получите уязвимость

**Comment by morfeusys** [comment]
@Merlin74 @seniorivn Безопасность настраивается все в том же aggregate.json - я опишу в документации. Пока можно [посмотреть тут](http://vertx.io/docs/vertx-shell)

**Comment by ivanbalashov239** [comment]
@Morfeusys ссылка битая? эта правильная?  http://vertx.io/docs/vertx-shell/java/

**Comment by morfeusys** [comment]
@seniorivn да, не ту ссылку прислал
